src/app/page.tsx

```diff
@@ -5,17 +5,22 @@ import { WorkflowStateResponse } from '@/types/dto/workflowStates.dto'
 import { apiUrl } from '@/config'
 import { ClientSideStateUpdate } from '@/hoc/ClientSideStateUpdate'
 import { TaskResponse } from '@/types/dto/tasks.dto'
+import { revalidateTag } from 'next/cache'
 
 async function getAllWorkflowStates(token: string): Promise<WorkflowStateResponse[]> {
-  const res = await fetch(`${apiUrl}/api/workflow-states?token=${token}`)
+  const res = await fetch(`${apiUrl}/api/workflow-states?token=${token}`, {
+    next: { tags: ['getAllWorkflowStates'] },
+  })
 
   const data = await res.json()
 
   return data.workflowStates
 }
 
 async function getAllTasks(token: string): Promise<TaskResponse[]> {
-  const res = await fetch(`${apiUrl}/api/tasks?token=${token}`)
+  const res = await fetch(`${apiUrl}/api/tasks?token=${token}`, {
+    next: { tags: ['getAllTasks'], revalidate: 0 },
+  })
```

Comment:
Please be sure to remove the `apiUrl` env key in the future after all the pending PRs are closed.

Reply:
Resolving